lua/cmp/config/default.lua

```diff
@@ -123,5 +116,18 @@ return function()
     },
 
     sources = {},
+
+    window = {
+      completion = {
+        border = {'', '', '', '', '', '', '', ''},
+        scrollbar = '',
+      },
+      documentation = {
+        border = {'', '', '', ' ', '', '', '', ' '},
+        maxheight = math.floor(WIDE_HEIGHT * (WIDE_HEIGHT / vim.o.lines)),
+        maxwidth = math.floor((WIDE_HEIGHT * 2) * (vim.o.columns / (WIDE_HEIGHT * 2 * 16 / 9))),
+        scrollbar = '',
```

Comment:
This is matter of my personal preference, but I would like to fix it to max_width / max_height.